x-pack/test/encrypted_saved_objects_api_integration/fixtures/api_consumer_plugin/server/index.ts

```diff
@@ -168,6 +168,7 @@ function defineTypeWithMigration(core: CoreSetup<PluginsStart>, deps: PluginsSet
             },
```

Comment:
Added functional test in https://github.com/elastic/kibana/pull/105968/commits/49f110cb92e3e2b2c444e33f9fa341f8a10c5a7b

I created an alert and action SO using a different encryption key than what is in the functional test, using an older version to make sure some of existing alert and action migrations run. When the functional test runs, you can see the decryption error throw and the warning get logged that the migration will run anyway. The test asserts that the alert and action SO have the new fields expected post-migration but retrieving them via `getDecryptedAsInternalUser` fails because the decryption fails. Do you think that's sufficient?

Reply:
Yes, sounds perfect, thanks for adding it! Would you mind leaving this clarification as a code comment in the test itself? That would help future readers to understand what we're validating there exactly.

Reply:
Comment added in https://github.com/elastic/kibana/pull/105968/commits/4711bd2a98deaae054fb331c07abce74cf8de821